pytplinkrouter/__init__.py

```diff
@@ -211,6 +213,53 @@ def _update_info(self):
         return False
 
 
+class TplinkArcherC20DeviceScanner(Tplink2DeviceScanner):
```

Comment:
Any reason you are inheriting from Tplink2DeviceScanner and not TplinkDeviceScanner like the others? Is it to have their version of get_device_name()? As your scanner is completely different from Tplink2, I think it would be cleaner to inherit from base one.